### PR TITLE
Feature/sm 2

### PR DESCRIPTION
**Pull request by andreeamuntean**

### CODE REVIEW

File: src/loose/oose/fis/student/manager/Main.java [hunk 1]
```diff
@@ -9,10 +9,11 @@
...
 
     @Override
-    public void start(Stage primaryStage) throws Exception{
-        Parent root = FXMLLoader.load(getClass().getResource("sample.fxml"));
-        primaryStage.setTitle("Hello World");
-        primaryStage.setScene(new Scene(root, 300, 275));
+
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
```
Comment (CosminaSas) [inline src/loose/oose/fis/student/manager/Main.java:15]: **“FIS - Student Manager”**

**Review by CosminaSas — COMMENTED** [review]
“SEF- Student Manager” please :)

File: src/loose/oose/fis/student/manager/Main.java (hunk 1 above)
Comment (andreeamuntean) [inline src/loose/oose/fis/student/manager/Main.java:15]: no =)

**Review by CosminaSas — APPROVED** [review]
meeeh.. approved

**Comment by CosminaSas** [comment]
if (username.equals("teacher") && password.equals("teacher")) {

   try {

       Stage stage = (Stage) loginMessage.getScene().getWindow();

       Parent viewStudentsRoot = FXMLLoader.load(getClass().getResource("../fxml/view-students.fxml"));

       Scene scene = new Scene(viewStudentsRoot, 600, 400);

       stage.setScene(scene);

   } catch (IOException e) {

       e.printStackTrace();

   }


   return;

}

**Pull request merged**